### ISSUE TITLE
Should we use Sigmoid instead Softmax?

### ISSUE THREAD

**Issue by rodrigo2019** [mkocabas/focal-loss-keras#9]
In the original paper the author said:

> We adopt this form in our experiments as it yields slightly
> improved accuracy over the non-α-balanced form. Finally,
> we note that the implementation of the loss layer combines
> the sigmoid operation for computing p with the loss computation,
> resulting in greater numerical stability.

using sigmoid as the last activation in the network for multi class problems means that has a possibility to get more than one high probability?
Why sigmoid got better stability than Softmax?

**Comment by mkocabas** [comment]
I think it has no theoretical roots. I often find sigmoid better than softmax in my experiments without a reasonable explanation.

**Comment by alyato** [comment]
@rodrigo2019 
Does i use this `loss` as `loss_function` for multi class?
Because you say `using sigmoid as the last activation in the network for multi class problems means that has a possibility to get more than one high probability`
It do means that i can use the `facal_loss` for multi class with `softmax`
And if i use the `sigmoid`not softmax for multi class, it will get much performance.
Is it right?
Thanks.
